articles/native-platforms/ios-objc/02-custom-login.md

```diff
@@ -158,25 +156,23 @@ You can use the `profile.name` to show the user's name or show the users profile
 If you are going to let the user sign in, you'll probably need to let the user to register first. And creating a user is as easy as sign in:
 
 ```objc
-    A0AuthenticationAPI *authApi = [[A0AuthenticationAPI alloc] initWithClientId:[Auth0InfoHelper Auth0ClientID] url:[Auth0InfoHelper Auth0Domain]];
-
-    [authApi signUpWithEmail:@"email@foo.com"
+A0AuthenticationAPI *authApi = [[A0AuthenticationAPI alloc] initWithClientId:[Auth0InfoHelper Auth0ClientID] url:[Auth0InfoHelper Auth0Domain]];
+[authApi signUpWithEmail:@"email@foo.com"
                     username:nil // by default the username is the users email
                     password:@"1234"
                   connection:@"Username-Password-Authentication"
                 userMetadata:@{ @"Country": @"Australia",
                                 @"Telephone": @"99 99 9999 9999"}
-                                // Here you can include any extra information
-                                // you need about the user
+                                 // Here you can include any extra information
+                                 // you need about the user
                        scope:@"openid"
                   parameters:nil
                     callback:^(NSError * _Nullable error, A0Credentials * _Nullable credentials) {
-            if(error) {
-                // Something went wrong, let the user know
-            } else {
-                // You signed up correctly, and you have the Credentials,
-                // so no need to make the user login. They are already in.
-            }
-        });
-}];
+                        if(error) {
+                            // Something went wrong, let the user know
+                        } else {
+                            // You signed up correctly, and you have the Credentials,
+                            // so no need to make the user login. He's already in.
```

Comment:
We try to stay gender neutral for users, so probably "They are already in" was accurate, or "The user is already in". Or just remove that last line "He's already in."